<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aboutDialog</class>
 <widget class="QDialog" name="aboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About gpsinfo4zemokost</string>
  </property>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Schlie√üen</string>
   </property>
   <property name="icon">
    <iconset theme="window-close">
     <normaloff/>
    </iconset>
   </property>
  </widget>
  <widget class="QLabel" name="aboutText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>600</width>
     <height>170</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
